osf/models/preprint_service.py

```diff
@@ -216,7 +216,7 @@ def set_identifier_values(self, doi, save=False):
             self.save()
 
     def get_doi_client(self):
-        if settings.CROSSREF_URL:
+        if settings.CROSSREF_URL and 'qatest' not in self.node.all_tags.values_list('name', flat=True):
```

Comment:
This should be able to be made into a query: `all_tags.filter(name='qatest').exists()`.